src/Sulu/Bundle/SecurityBundle/Security/Exception/EmailNotUniqueException.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\SecurityBundle\Security\Exception;
+
+/**
+ * This Exception is thrown if the email for a user is not unique
+ * @package Sulu\Bundle\SecurityBundle\Security\Exception
+ */
+class EmailNotUniqueException extends SecurityException
+{
+    public function __construct()
```

Comment:
Please add the non-unique email to the constructor and implement a getter for it, so that somebody could use this information in a catch block.